### PR TITLE
Move all aliases into their own file.

### PR DESCRIPTION
**Pull request by nbp**
###### Things done:
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`

This patch just move a sub-part of all-packages into its own file.  I am separating this change from #14000, because it will bitrot quickly and force the rebase of the changes everytime.

This work is needed for making seamless security updates (#10851), as we would have to replace the `aliases` variable by another one taken from another tree.

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -16464,92 +16464,7 @@ let
...
-  vimbWrapper = vimb; # added 2015-01
-  vimprobable2Wrapper = vimprobable2; # added 2015-01
-  virtviewer = virt-viewer; # added 2015-12-24
-  vorbisTools = vorbis-tools; # added 2016-01-26
-  x11 = xlibsWrapper; # added 2015-09
-  xf86_video_nouveau = xorg.xf86videonouveau; # added 2015-09
-  xlibs = xorg; # added 2015-09
-  youtubeDL = youtube-dl;  # added 2014-10-26
-};
+aliases = import ./alias-packages.nix self;
```
Comment (zimbatm) [inline pkgs/top-level/all-packages.nix]: why not just `aliases.nix` ? naming :)

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -16464,92 +16464,7 @@ let
 }; # self_ =
 
 
-  ### Deprecated aliases - for backward compatibility
```
Comment (zimbatm) [inline pkgs/top-level/all-packages.nix]: I think we do want to keep this comment or similar in this file. It makes it easier for new-comers to find the right file.

**Comment by zimbatm** [comment]
LGTM otherwise

**Comment by nbp** [comment]
I will apply the comments, and merge the patch.

**Comment by jagajaga** [comment]
cc @edolstra

**Pull request merged**